### DISCUSSION TITLE
Widgets set to hidden in QGIS are not hidden in qfield

### DISCUSSION THREAD

**Discussion by bladnor** [opengisch/QField#5919] [Q&A]
By now I have tried a hundered things but can not manage to make a layer field widget to hide in the form. In QGIS it works but when I copy the project to qfield the widgets marked as hidden are still shown on the form!?
What am I doing possibly wrong? What can I check to find the problem? I'm running out of ideas.
Tested with these version:
QField Version: 3.4.7 - Ebo
QGIS Version: tried with 3.40 and 3.34.6
Thanks Roland

**Comment by nirvn** [comment]
@bladnor , can you provide a sample project? Conditional visibility of fields work just fine over here (you can see it working with the sample bees project too with the apiary's infected feature form tab)
